scripts/install-chef-suse.sh

```diff
@@ -778,13 +778,12 @@ fi
 # Take care that the barclamps are installed in the right order (as expressed
 # in cookbook dependencies)
 #
-required_barclamps="crowbar deployer dns ipmi logging network ntp provisioner"
+required_barclamps="core"
```

Comment:
IIUC, this variable name is no longer correct.  `core` is not a barclamp - it's a group of barclamps.  Right?  Same with `openstack`.